sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -5,6 +5,7 @@
     <PackageTags>Azure;Event Hubs;EventHubs;.NET;AMQP;IoT;$(PackageCommonTags)</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <EnableFxCopAnalyzers>false</EnableFxCopAnalyzers>
+    <RunApiCompat>false</RunApiCompat>
```

Comment:
I think anywhere we disable the apicompat tool we should probably file a bug to get it re-enabled otherwise it likely be forgotten.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/9586
https://github.com/Azure/azure-sdk-for-net/issues/9585
https://github.com/Azure/azure-sdk-for-net/issues/9584
https://github.com/Azure/azure-sdk-for-net/issues/9579